spec/ic.did

```diff
@@ -52,8 +52,8 @@ type get_utxos_request = record {
   address : bitcoin_address;
   network: bitcoin_network;
   filter: opt variant {
-    min_confirmations: nat32;
-    page: blob;
+    MinConfirmations: nat32;
```

Comment:
That seems inconsistent with the naming conventions. Maybe we should actually fix the implementation instead? WDYT @ielashi @THLO